toplevel/native/topmain.ml

```diff
@@ -49,26 +49,33 @@ let prepare ppf =
       false
 
 let file_argument name =
```

Comment:
(Same naming question on `input_argument`. There is some unfortunate duplication among the bytecode and native toplevels.)